src/main/java/de/srendi/advancedperipherals/common/util/fakeplayer/APFakePlayer.java

```diff
@@ -64,7 +67,7 @@ public class APFakePlayer extends FakePlayer {
     private float currentDamage = 0;
 
     public APFakePlayer(ServerLevel world, Entity owner, GameProfile profile) {
-        super(world, profile != null && profile.isComplete() ? profile : PROFILE);
+        super(world, PROFILE);
```

Comment:
I don't quite understand this change, would you mind to explain a bit why not using player profile anymore?
I think if we use player profile then chunk claim stuffs can working with correct permissions.

Reply:
I removed that for testing instead of commenting it out and placing a //TODO next to it and I completely forgot about it. The initial reason why I removed that was because `isComplete` was removed. But that is not necessary anymore, the relevant checks seem to be now in the constructor. Except for the string blank check for the name, but I can just check that ourselves.

That happens when you make a PR/Port with a half year pause in between